src/extension.ts

```diff
@@ -535,7 +535,7 @@ function isDecryptedFile(uri: vscode.Uri) {
 function getDecryptedFileUri(encryptedUri: vscode.Uri): vscode.Uri {
 	const decryptedFileName = DECRYPTED_PREFIX + path.basename(encryptedUri.path);
 	const decryptedFilePath = path.join(path.dirname(encryptedUri.path), decryptedFileName);
-	const decryptedFileUri = encryptedUri.with({ path: decryptedFilePath });
+	const decryptedFileUri = encryptedUri.with({ path: decryptedFilePath.replace(/\\/g, "/") });
```

Comment:
Maybe the more stable would be using some node.js path helper for "standardizing"/"converting" path from windows format to posix (unix) format. There are some helpers for that.